.circleci/config.yml

```diff
@@ -133,6 +133,56 @@ jobs:
     steps:
       *test_steps
 
+  ruby25rails60beta:
+    docker:
+    - image: circleci/ruby:2.5.1
+
+    environment:
+      BUNDLE_GEMFILE: test/gemfiles/Gemfile-Rails-6-0-beta
+
+    steps:
+      *test_steps
```

Comment:
This job is already in master. Can you make sure this PR is properly rebased?

Reply:
I alread rebased after the previous PR was merged.

Reply:
I see that now, Github is showing the merged commit as part of this PR. I'm :astonished:.

Reply:
Can you try squashing commits to see if it fixes the diff? Feel free to leave the addition of the forgotten jruby + 2.5 entry as a separate commit.

Reply:
A easy way is to cancel this PR and open another with the correct way

Reply:
Yeah, no sure what happened here... The other PR looks great! :+1: